test/test-status.sh

```diff
@@ -108,6 +108,13 @@ begin_test "status --json"
 
   expected='{"files":{"file2.dat":{"status":"R","from":"file1.dat"}}}'
   [ "$expected" = "$(git lfs status --json)" ]
+
+  git commit -m "file1.dat -> file2.dat"
+
+  echo hi > test1.txt
```

Comment:
Thank you for adding this test case! Would you mind either:

- Adding a `#` comment above saying what this is testing, or...
- Extracting this out to a separate test?